.env

```diff
@@ -26,8 +26,8 @@ IPL_POSTGRES_PORT=5432
 IPL_POSTGRES_DB=geoserver
 IPL_POSTGRES_USER=geoserver
 
-# x2gbfs variables
-X2GBFS_IMAGE=ghcr.io/mobidata-bw/x2gbfs:2024-04-22T11-45
+# x2gbfs variables (Note: when changing the providers, be sure to check if the image version supports them)
```

Comment:
This is ok, but we need to define a process for synchronizing such activities related to versioning.